src/commands/tick-login.js

```diff
@@ -27,9 +26,9 @@ function login(argv) {
 
     if (err) 
       throw err
-    
+ 
     server.login(user)
-    .then(user => setConfig('remote', user.couch.url))
+    .then(user => account.setUser(user))
```

Comment:
Should we provide feedback to the user that their user id was added to N documents?